<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="sendFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>1041</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>1001</width>
       <height>90</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="save_video_button">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
        </property>
        <property name="text">
         <string>Save the video</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>471</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
     </property>
     <property name="text">
      <string>Enter an encryption key:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditKey">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>150</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="upload_file_button">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
        </property>
        <property name="text">
         <string>Upload a txt file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>1001</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="generate_video_button">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
        </property>
        <property name="text">
         <string>Generate video</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEditFile">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
    </property>
    <property name="text">
     <string>Processing...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1031</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
       </property>
       <property name="text">
        <string>Video Steganography</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEditVideo">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>481</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="upload_video_button">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
       </property>
       <property name="text">
        <string>Upload video</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="recieveFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>1041</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>1001</width>
       <height>90</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="save_message_button">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
        </property>
        <property name="text">
         <string>Save the message</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>471</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
     </property>
     <property name="text">
      <string>Enter the decryption key:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditKey2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>1001</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QPushButton" name="extract_message_button">
        <property name="styleSheet">
         <string notr="true">font: 15pt &quot;Papyrus&quot;;
color: &quot;black&quot;;</string>
        </property>
        <property name="text">
         <string>Extract message</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSend_or_recieve">
    <property name="title">
     <string>Send or recieve?</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSend"/>
    <addaction name="actionRecieve"/>
   </widget>
   <addaction name="menuSend_or_recieve"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSend">
   <property name="text">
    <string>Send</string>
   </property>
  </action>
  <action name="actionRecieve">
   <property name="text">
    <string>Recieve</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
